Constrained_triangulation_3/doc/Constrained_triangulation_3/Constrained_triangulation_3.txt

```diff
@@ -0,0 +1,224 @@
+Left: PLC (360 vertices);
+Middle: CCDT (2452 vertices);
+Right: the same CCDT seen with cutplane.
+\cgalFigureCaptionEnd
+
+
+\section CT_3_api Software Design
+
+\subsection CT_3_api_classes Classes
+
+This package provides one main class `CGAL::Conforming_constrained_Delaunay_triangulation_3`
+that represents a 3D conforming constrained Delaunay triangulation.
+The class is templated by the geometric traits class and the underlying triangulation class.
+The other classes that are provided are secondary classes that define the
+vertex and cell types and metadata that constitute the triangulation.
+
+\subsection CT_3_api_functions Functions
+
+Some helper constructor functions, like `CGAL::make_conforming_constrained_Delaunay_triangulation_3()`,
+are provided to create a `CGAL::Conforming_constrained_Delaunay_triangulation_3` object.
+
+\section CT_3_examples Examples
+
+\subsection CT_3_example_ccdt Build a Conforming Constrained Delaunay Triangulation
+
+The following example demonstrates how to use the helper constructor function
+`CGAL::make_conforming_constrained_Delaunay_triangulation_3()` to create a
+conforming constrained Delaunay triangulation from a given PLC.
+
+\cgalExample{Constrained_triangulation_3/conforming_constrained_Delaunay_triangulation_3.cpp }
+
+
+\subsection CT_3_example_ccdt_soup Build a Conforming Constrained Delaunay Triangulation from a Polygon Soup
+
+It is also possible to create a conforming constrained Delaunay triangulation
+from a polygon soup, i.e., a set of polygons for which the connectivity is not known a priori.
+This example demonstrates how to build such a triangulation.
+
+\cgalExample{Constrained_triangulation_3/conforming_constrained_Delaunay_triangulation_3_from_soup.cpp }
+
+
+\subsection CT_3_example_ccdt_fpmap Build a Conforming Constrained Delaunay Triangulation with Known Face Patches
+
+When the user knows a priori the set of patch ids that he wants to attach to faces, this information
+can be used and kept valid through the construction of the conforming constrained
+Delaunay triangulation.
+
+The following example demonstrates how to detect surface patches separated by sharp edges
+and use this surface segmentation in the tetrahedrization process.
+
+When this parameter is used, the 3D triangulation constrained faces indices (available with
+`CGAL::Conforming_constrained_Delaunay_triangulation_cell_data_3::face_constraint_index() `
+)
+are set to the corresponding patch ids from this map.
+When it is not used, the face indices are set to different indices for each input polygon.
+
+\cgalExample{Constrained_triangulation_3/conforming_constrained_Delaunay_triangulation_3_fpmap.cpp}
+
+\cgalFigureRef{ccdt_fpmap} shows the input and output of this triangulation construction example.
+
+\cgalFigureBegin{ccdt_fpmap, ccdt_fpmap.png}
+Left: Input PLC;
+Middle-left: The corresponding conforming constrained Delaunay triangulation with one patch per input polygon;
```

Comment:
you have Middle-left but Middle-Right. And I'm not sure that Middle-Left/Right are correct anyway.

Reply:
Jane, I do not know what to do with this comment. Should we add IDs (a) (b) (c) (d) to the
sub-figures?

Reply:
I would add "from left to right : ..."